src/utils/test/simple-gu-stack.ts

```diff
@@ -1,21 +1,11 @@
 import { App } from "@aws-cdk/core";
-import type { Environment } from "@aws-cdk/core";
-import { GuStack, GuStackForInfrastructure } from "../../constructs/core";
+import { GuStack } from "../../constructs/core";
 import type { GuStackProps } from "../../constructs/core";
 
-// Some stacks (such as ones using access logging on load balancers) require specifying a region
-interface SimpleGuStackProps extends Partial<GuStackProps> {
-  env?: Environment;
-}
-
-export const simpleGuStackForTesting: (props?: SimpleGuStackProps) => GuStack = (props?: SimpleGuStackProps) =>
-  new GuStack(new App(), "Test", {
-    stack: props?.stack ?? "test-stack",
-    ...props,
-  });
-
-export const simpleInfraStackForTesting: (props?: SimpleGuStackProps) => GuStack = (props?: SimpleGuStackProps) =>
-  new GuStackForInfrastructure(new App(), "Test", {
+export function simpleGuStackForTesting(props?: Partial<GuStackProps>) {
```

Comment:
Switching the a `function` rather than a `const function` as the type annotations are simpler.